nexus-repository-r-it/src/test/java/org/sonatype/nexus/plugins/r/internal/RHostedIT.java

```diff
@@ -80,7 +80,6 @@ public void testUploadFailedWrongPackageExtension() throws Exception
   {
     HttpResponse httpResponse = uploadSinglePackage(AGRICOLAE_131_XXX);
     assertThat(httpResponse.getStatusLine().getStatusCode(), is(BAD_REQUEST));
```

Comment:
we need to investigate the problem. Please create a technical debt ticket

Reply:
https://issues.sonatype.org/browse/NEXUS-24838